docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Edge multi-cluster"
+linkTitle: "Edge multi-cluster"
+weight: 1
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a state-based management platform to declaratively developing and operating workloads. Kube allows a developer to
+ create workloads that are node-agnostic and can be constrained to operate in places where certain conditions are met. Operators enjoy
+  gaurantee that workloads are rescheduled to work wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
+There is a growing community of cloud native developers and operators that work for organizations that require more fine-grain control
+ over the distribution of workload due to the nature of their business. Telecommunications, industrial manufacturers, and automotive
+  companies would like to rely on Kubernetes state-based management but they need more flexibility to accomodate for conditions like
+   large-scale deployments, disconnected and autonomous operation, and constrained cluster size. These requirements represent the Edge 
```

Comment:
large-scale deployments is not really a "condition" , it's more of a requirement that impact the underlying solution/platform

Reply:
I can change the term "conditions" to "requirements" if you like.

Reply:
changed to 'large number of edge locations'